.travis.yml

```diff
@@ -69,7 +67,7 @@ before_install:
   - yarn global add greenkeeper-lockfile@1
 
 install:
-  - yarn install --non-interactive
+  - yarn install --non-interactive --frozen-lockfile
```

Comment:
I believe for addons you want to *not* frozen-lockfile. The idea is that when a consuming app installs your dependencies, they get the latest matching versions not the contents of your lockfile. Thus to really test for those consuming cases, you need let the dependencies float.

It does mean your devDeps float in addition to your dependencies, however.